storage/include/storage/test/s3_test_common.h

```diff
@@ -127,7 +127,7 @@ class S3Test : public ::testing::Test {
   struct shared_state {
     concord::storage::s3::StoreConfig config;
     fs::path test_dir;
-    std::unique_ptr<concord::storage::s3::Client> client;
+    std::shared_ptr<concord::storage::s3::Client> client;
```

Comment:
Wasn't the binary which used this test removed from CMakeLists?
Assuming that this is still being used, why the switch to a shared_ptr?

Reply:
shared_ptr to s3::Client is held by s3::Client::Trnsaction objects

Reply:
So this previously  being a unique ptr was a bug?